gen/definitions/host.yaml

```diff
@@ -12,10 +12,8 @@ attributes:
   - model_name: type
     type: String
     description: Type of the object; this value is always 'Host'.
-    default_value: Host
-    requires_replace: true
+    read_only: Host
     exclude_example: true
```

Comment:
`exclude_example` should not be required for read-only attributes. If necessary, we should update the templates rendering the examples to always exclude those attributes.

Reply:
changed